Doc/whatsnew/3.12.rst

```diff
@@ -70,6 +70,17 @@ Important deprecations, removals or restrictions:
 
 * :pep:`623`, Remove wstr from Unicode
 
+Improved Error Messages
+=======================
+
+* Improve the :exc:`SyntaxError` error message when the user types ``import x
+  from y`` instead of ``from y import x``. Contributed by Pablo Galindo in :gh:`98931`.
+
+    >>> import a.y.z from b.y.z
```

Comment:
```suggestion
    >>> import a.y.z from b.y.z
    Traceback (most recent call last):
```
Fix doctest. Does this change meet your expectations? I noticed that there are similar cases in unit tests, so I didn't omit the traceback stack here to show more exception information.